src/core/lib/iomgr/systemd_utils.cc

```diff
@@ -36,8 +36,30 @@ bool set_matching_sd_unix_fd(grpc_tcp_server* s,
                              const grpc_resolved_address* addr,
                              const int fd_start, const int n) {
   absl::StatusOr<std::string> addr_name = grpc_sockaddr_to_string(addr, true);
+  if (!addr_name.ok()) {
+    return false;
+  }
+  std::string &address = addr_name.value();
```

Comment:
You can just use `addr_name->...` to avoid having two names for the same value. See https://abseil.io/tips/181.